data/laps/lap1-0.csv

```diff
@@ -1,2304 +1,2304 @@
-48.0494257,17.568047099999998,14.11,199.93119000000002,48.0494141,17.568040099999997,201.82661000000002
-48.0494141,17.568040099999997,13.84,201.82661000000002,48.049402799999996,17.5680331,201.9027
-48.049402799999996,17.5680331,13.64,201.9027,48.049391899999996,17.5680257,202.74693000000002
-48.049391899999996,17.5680257,13.31,202.74693000000002,48.0493813,17.5680182,204.84342
-48.0493813,17.5680182,13.01,204.84342,48.049371199999996,17.5680102,205.56118
-48.049371199999996,17.5680102,12.72,205.56118,48.049361499999996,17.568002099999998,208.08314000000001
-48.049361499999996,17.568002099999998,12.38,208.08314000000001,48.0493522,17.567994,209.60170000000002
-48.0493522,17.567994,11.99,209.60170000000002,48.0493434,17.567985699999998,210.80239000000003
-48.0493434,17.567985699999998,11.74,210.80239000000003,48.0493348,17.567977499999998,212.30985
-48.0493348,17.567977499999998,11.34,212.30985,48.0493266,17.5679694,212.30985
-48.0493266,17.5679694,11.34,212.30985,48.049318799999995,17.5679613,214.04856
-48.049318799999995,17.5679613,10.64,214.04856,48.0493113,17.5679533,214.80193000000003
-48.0493113,17.5679533,10.36,214.80193000000003,48.049304299999996,17.5679452,216.26962000000003
-48.049304299999996,17.5679452,9.96,216.26962000000003,48.049297599999996,17.567937399999998,218.07743000000002
-48.049297599999996,17.567937399999998,9.61,218.07743000000002,48.0492914,17.567929499999998,217.85537000000002
-48.0492914,17.567929499999998,9.22,217.85537000000002,48.049285399999995,17.567921899999998,220.17716000000001
-48.049285399999995,17.567921899999998,8.83,220.17716000000001,48.0492797,17.5679147,220.17716000000001
-48.0492797,17.5679147,8.83,220.17716000000001,48.049274399999995,17.567907599999998,220.65916
-48.049274399999995,17.567907599999998,8.07,220.65916,48.0492693,17.5679011,221.55107
-48.0492693,17.5679011,7.76,221.55107,48.0492645,17.5678947,220.85248
-48.0492645,17.5678947,7.32,220.85248,48.049259899999996,17.567888699999997,221.72655000000003
-48.049259899999996,17.567888699999997,6.91,221.72655000000003,48.049255699999996,17.5678832,220.65632000000002
-48.049255699999996,17.5678832,6.61,220.65632000000002,48.0492516,17.567878,220.74945000000002
-48.0492516,17.567878,6.07,220.74945000000002,48.0492479,17.5678731,221.10654000000002
-48.0492479,17.5678731,5.82,221.10654000000002,48.0492443,17.5678686,221.55495000000002
-48.0492443,17.5678686,5.3,221.55495000000002,48.049241099999996,17.5678646,220.3692
-48.049241099999996,17.5678646,5.0,220.3692,48.0492381,17.5678608,219.94348000000002
+LAT,LON,GSPEED,CRS,NLAT,NLON,NCRS,CMD
+48.0492306,17.567850699999997,0.0,0.0,48.049231299999995,17.567849399999997,0.0,11.0116384005144
+48.049231299999995,17.567849399999997,0.04,0.0,48.0492305,17.5678495,0.0,-12.070141112379401
+48.0492305,17.5678495,0.01,0.0,48.0492292,17.5678499,0.0,-19.6874865336079
+48.0492292,17.5678499,0.01,0.0,48.0492283,17.5678498,0.0,-13.9127673190133
+48.0492283,17.5678498,0.02,0.0,48.0492274,17.56785,0.0,-13.781592113715
```

Comment:
For example, here

Reply:
I really have no idea, I think I've already seen it happen in some cases but I don't know why

Reply:
Ideally, we need to recreate the file differences.cvs.
However, it seems that differences are neglectable.